src/main/java/seedu/duke/model/template/LcTemplate.java

```diff
@@ -100,7 +100,8 @@ public void setComponent(String s, double value) {
      */
     @Override
     public String toString() {
-        return LC_TEMPLATE + "Total Capacitance: " + capacitor + System.lineSeparator()
+        return "Current Voltage: " + initialPowerSupply + System.lineSeparator()
+                + LC_TEMPLATE + "Total Capacitance: " + capacitor + System.lineSeparator()
```

Comment:
Perhaps could have added Current Voltage printed after the template?

Reply:
Consider putting this in CircuitTemplate instead? Then subclasses can just call `super.toString()`.